packages/tools/devtools/devtools-core/src/IFluidDevtools.ts

```diff
@@ -33,6 +34,15 @@ export interface IFluidDevtools extends IDisposable {
 	 */
 	registerContainerDevtools(props: ContainerDevtoolsProps): void;
 
+	/**
+	 * Registers a data object with the devtools.
+	 *
+	 * @remarks Although the data object is NOT a container, it is treated as one and stored in `containers` map for the purposes of the devtools.
+	 *
+	 * @throws Will throw if devtools have already been registered for the specified id generated by {@link @fluidframework/datastore-definitions#IFluidDataStoreContext.generateDocumentUniqueId}.
```

Comment:
Nit:
```suggestion
	 * @throws Will throw if devtools have already been registered for the provided data object.
```

Reply:
The way we generate IDs should probably stay an implementation detail. The user only cares that this will throw if they pass the same data object more than once.